spec.md

```diff
@@ -891,7 +892,10 @@ reg myreg: SInt, myclock
 Optionally, for the purposes of circuit initialization, a register can be
 declared with a reset signal and value. In the following example,
 `myreg`{.firrtl} is assigned the value `myinit`{.firrtl} when the signal
-`myreset`{.firrtl} is high.
+`myreset`{.firrtl} is high.  The reset signal must be a `Reset`{.firrtl},
+`UInt<1>`{.firrtl}, or `AsyncReset`{.firrtl}, and the type of initialization 
+value must be equivalent to the declared type of the register 
+(see [@sec:type-equivalence] for details).
```

Comment:
SFC requires the reset to be `UInt<1>`, but MFC will accept an unknown width `UInt` that must infer to a `UInt<1>`. It seems fine to start with the weaker MFC interpretation here.

Reply:
This specifies Uint<1>